lib/rubocop/cop/lint/debugger.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'set'
```

Comment:
No need to `require 'set'` here, it's required by `parser` and (just to be explicit) by `rubocop-ast` too.